scripts/retrigger_file_validation/usage.md

```diff
@@ -0,0 +1,15 @@
+# How to revalidate failed file validation in Ingest
```

Comment:
Can you link to the relevant ticket please?

Reply:
Done